README.md

```diff
@@ -22,8 +22,10 @@ are contained in the 'res' directory.
 The following programs and/or libaries need to be installed before compiling
 and installing the RMA:
 
-Asterisk (>= version 13.0.0)
-libcurl
+- Asterisk (>= version 13.0.0)
+- libcurl
+- pjproject
+- res_rtp_asterisk
```

Comment:
Hopefully someone can correct me if I am wrong here, but I believe that you can build res_rtp_asterisk without pjproject. However, the functionality of that module would be limited and there would be no support for things like ICE (which of course is needed for respoke). This would apply for both chan_sip or chan_pjsip.